pdns/slavecommunicator.cc

```diff
@@ -618,6 +610,34 @@ void CommunicatorClass::suck(const DNSName &domain, const ComboAddress& remote,
       notifyDomain(domain, &B);
     }
 
+    // Execute LUA-AXFR-END-SCRIPT script
+    unique_ptr<AuthLua4> laes{nullptr};
+    vector<string> axfr_end_scripts;
+    string axfr_end_script=::arg()["lua-axfr-end-script"];
+    if(B.getDomainMetadata(domain, "LUA-AXFR-END-SCRIPT", axfr_end_scripts) && !axfr_end_scripts.empty()) {
+      if (pdns_iequals(axfr_end_scripts[0], "NONE")) {
+        axfr_end_script.clear();
+      } else {
+        axfr_end_script=axfr_end_scripts[0];
+      }
+    }
+    if(!axfr_end_script.empty()){
+      try {
+        laes = make_unique<AuthLua4>();
+        laes->loadFile(axfr_end_script);
+        g_log<<Logger::Info<<logPrefix<<"loaded Lua script '"<<axfr_end_script<<"'"<<endl;
```

Comment:
again, this might be too verbose.

Reply:
https://github.com/m0rcq/pdns/commit/d120e5831f53ebdc6e84aae291e008cfaf390fa4 sets it down to debug log level to reduce verbosity in normal operations.